### PR TITLE
Making it easier to get pre-signed head requests

### PR DESCRIPTION
**Pull request by mrloop**
Very similar to get_url. https://github.com/fog/fog-aws/issues/47

### CODE REVIEW

File: lib/fog/aws/requests/storage/head_object_url.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+module Fog
+  module Storage
+    class AWS
+      module HeadObjectUrl
+        def head_object_url(bucket_name, object_name, expires, options = {})
```
Comment (mrloop) [inline lib/fog/aws/requests/storage/head_object_url.rb]: Wondering wither this and duplicate code in get_object_url should be put in module?

Comment (geemus) [inline lib/fog/aws/requests/storage/head_object_url.rb]: Possibly, but it is a small enough amount of code I'm not too concerned about it for now anyway.

**Comment by geemus** [comment]
Thanks!

**Comment by mrloop** [comment]
Great, hopefully it'll be useful to somebody else.

**Pull request merged**